docs/book/src/developer/providers/v1alpha2-to-v1alpha3.md

```diff
@@ -52,13 +52,24 @@
 ## Changes to `sigs.k8s.io/cluster-api/controllers/remote`
 
 -  The `ClusterClient` interface has been removed.
-- `remote.NewClusterClient` now returns a `sigs.k8s.io/controller-runtime/pkg/client` Client. The signature changed from
+- `remote.NewClusterClient` now returns a `sigs.k8s.io/controller-runtime/pkg/client` Client. It also requires `client.ObjectKey` instead of a cluster reference. The signature changed from
```

Comment:
```suggestion
- `remote.NewClusterClient` now returns a `sigs.k8s.io/controller-runtime/pkg/client` Client. It also requires `client.ObjectKey` instead of a cluster reference. The signature changed:
```